test/dlc_tests/container_tests/bin/testDataserviceDistribute

```diff
@@ -0,0 +1,8 @@
+HOME_DIR=/test
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs
+
+python ${BIN_DIR}/test_dataset_service.py --local=False --distribute=True|| exit 1
+
+exit 0
+
```

Comment:
Please place `#!/bin/bash` at the beginning of the script to define how this script should executed, and insert `set -ex` in the beginning of the script to ensure that:
1. "-e": the script exits immediately if a command fails with a non-zero exit code
2. "-x": each command executed is printed to console

Reply:
Updated

Reply:
nit: You can now remove the `|| exit 1` on line 9.